extensions/ql-vscode/test/vscode-tests/no-workspace/query-history/query-history-manager.test.ts

```diff
@@ -228,25 +220,6 @@ describe("QueryHistoryManager", () => {
         });
       });
     });
-
-    describe("double click", () => {
-      it("should do nothing", async () => {
-        queryHistoryManager = await createMockQueryHistory(allHistory);
-        const itemClicked = allHistory[0];
-        const secondItemClicked = allHistory[1];
-
-        await queryHistoryManager.handleItemClicked(itemClicked, [
-          itemClicked,
-          secondItemClicked,
-        ]);
-
-        expect(localQueriesResultsViewStub.showResults).not.toHaveBeenCalled();
-        expect(variantAnalysisManagerStub.showView).not.toBeCalled();
-        expect(
-          queryHistoryManager.treeDataProvider.getCurrent(),
-        ).toBeUndefined();
-      });
-    });
```

Comment:
Has the behaviour changed or is there a different reason for removing this test?

I can see we've removed some of the tests that were checking for multiple items being passed, when the function should only receive a single item. I think for those cases, it makes sense for the tests to become redundant.

Reply:
I'm deleting this test because it was covering the case of having multiple items selected. I don't quite understand why it's named "double click", but in practice it's calling `handleItemClicked` as if two items in the query history are selected, and then asserting it ignores it and does nothing. It's now no longer possible to pass two items to `handleItemClicked`, so we don't need this test anymore.

Reply:
> It's now no longer possible to pass two items to handleItemClicked, so we don't need this test anymore.

Ah, I see. Thanks for explaining!

Reply:
_I think_ the reason we added the test in the first place was because double clicking was causing problems and we fixed the issue. We wanted to make sure we account for the behaviour. It was kind of a superfluous test if we were already testing this in other places. Perhaps it was me not having a proper overview of the existing tests when this was introduced. Anyway, I agree we don't need this.